app/views/admin/payroll_runs/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-xl">
+      Payroll
+    </h1>
+  </div>
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">Make a payroll file containing all the relevant information for approved claims to be paid by a payroll provider.</p>
```

Comment:
I think this text should have more of a focus on "preparing" the month's payroll. We're not just generating a file, we're actively assigning a set of claims to be processed as part of the month's payroll. I worry that the way it currently reads it makes it sound like a just building a file the same way the previous download link worked. I'll have a think and suggest some alternative words if you like?

Reply:
If you can come up with some content so we can get this in that would be great.

Reply:
For the sake of getting this out in time to run a proper payroll run for October, let's get this out as-is. We can come back and adjust the words as a separate piece of work.